package.json

```
@@ -1,14 +1,28 @@
 {
   "name": "postcss-rename",
-  "version": "0.7.1",
-  "description": "A PostCSS plugin to replace class names based on a customizable renaming scheme.",
+  "version": "0.8.0-dev",
+  "description": "A PostCSS plugin to replace CSS names based on a customizable renaming scheme.",
   "keywords": [
     "postcss",
     "css",
     "postcss-plugin"
   ],
-  "main": "build/src/index.js",
-  "types": "build/src/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./build/src/index.js",
+      "import": "./build/src/index.js",
+      "types": "./build/src/index.d.ts"
+    },
+    "./variable": {
+      "require": "./build/src/variable.js",
+      "import": "./build/src/variable.js",
+      "types": "./build/src/variable.d.ts"
+    },
+    "./types": {
+      "require": "./build/src/types.d.ts"
+    }
+  },
+  "types": "build/src/types.d.ts",
```

Remove "imports", change "types" path to "build/types.d.ts"

```suggestion
  "exports": {
    ".": {
      "require": "./build/src/index.js",
      "types": "./build/src/index.d.ts"
    },
    "./variable": {
      "require": "./build/src/variable.js",
      "types": "./build/src/variable.d.ts"
    },
  },
  "types": "build/types.d.ts",
```

---

Hmm, when I tested (with just js since TS still doesn't resolve types properly) it didn't work without import; I will double check tomorrow though